test/case/ietf_interfaces/dual_bridge/test.py

```diff
@@ -7,7 +7,7 @@
 """
 Dual bridges
 
-Test that it is possible to ping a bridge through a bridge to another bridge with VETH interfaces.
+Test that it is possible to ping through a bridge to another bridge via VETH interfaces.
```

Comment:
I prefer "Verify" over "Test", but pick what you like :)

Reply:
Maybe something like:
"Verify that it is possible to ping through one bridge to another bridge using VETH interfaces"

Reply:
Fixed